mall-portal/src/test/java/com/macro/mall/portal/util/CharUtilTest.java

```diff
@@ -0,0 +1,49 @@
+package com.zscat.mall.portal.util;
+
+import static org.mockito.Matchers.anyInt;
+
+import com.zscat.mall.portal.util.CharUtil;
+import java.util.Random;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+public class CharUtilTest {
+  @PrepareForTest({CharUtil.class})
+  @Test
+  public void testGetRandomNum() throws Exception {
+    final Random random = PowerMockito.mock(Random.class);
+    PowerMockito.when(random.nextInt(anyInt())).thenReturn(8);
+    PowerMockito.whenNew(Random.class).withNoArguments().thenReturn(random);
+
+    Assert.assertEquals("8", CharUtil.getRandomNum(1));
+    Assert.assertEquals("", CharUtil.getRandomNum(0));
+  }
+
+  @PrepareForTest({CharUtil.class})
+  @Test
+  public void testGetRandomString() throws Exception {
+    final Random random = PowerMockito.mock(Random.class);
+    PowerMockito.when(random.nextInt(anyInt())).thenReturn(32);
+    PowerMockito.whenNew(Random.class).withNoArguments().thenReturn(random);
+
+    Assert.assertEquals("6", CharUtil.getRandomString(1));
+    Assert.assertEquals("", CharUtil.getRandomString(0));
+  }
+
+  @Test
+  public void testPadLeft() {
+    Assert.assertEquals("a\'b\'c\u0000", CharUtil.padLeft("a\'b\'c", 6, '\u0000'));
```

Comment:
line > 80 chars

Reply:
Done. Even if I think this 80 char restriction is pointless.